packages/frontend/src/ee/pages/AiAgents/AiAgentNewThreadPage.tsx

```diff
@@ -36,7 +36,21 @@ const AiAgentNewThreadPage: FC = () => {
     );
     const { data: modelOptions } = useModelOptions({ projectUuid, agentUuid });
 
-    const [selectedModelId, setSelectedModelId] = useState<string | null>(null);
+    const [selectedModelId, _setSelectedModelId] = useState<string | null>(
```

Comment:
idg why the state setter is underscored. Also, we don't need to add setStates as React deps because they're stable always